File: POO_languages/JAVA/Car.java
public class Car {
    Integer id;
    String license;
    Account driver;
    Integer passenger;

    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;
    }

    void printData() {
        System.out.println("License: " + license + " Driver Name: " + driver.name);
    }
}

File: POO_languages/JS/car.js
class Car {
    id;
    license;
    driver;
    passenger;

    constructor(license, driver) {
        this.license = license;
        this.driver = driver;
    }

    printDataCar() {
        console.log(this.license);
        console.log(this.driver.name);
        console.log(this.driver.document);
    }
}